server/src/main/java/com/jungleapp/cs414/server/Piece.java

```diff
@@ -67,7 +67,7 @@ public String getPosition(){
         }
 
         //check down
-        if (row < 8) {
+        if (row < 9) {
```

Comment:
Will this not cause an  out of bounds? The index for row are zero to eight  so if the index is eight (which is less than nine) and you add one to it to check the space below it you would get an index out of bounds since there is no index nine.